lib/reverse_words.rb

```diff
@@ -1,4 +1,61 @@
-# A method to reverse each word in a sentence, in place.
+# # A method to reverse each word in a sentence, in place.
+require 'pry'
+
+def reverse_one_word(word, start = 0, finish = word.length)
+  if word == nil
+    return nil
+  end
+
+  i = start
+  index = finish
+
+  while i < index
+    first_letter = word[i]
+    last_letter = word[index]
+
+    word[i] = last_letter
```

Comment:
Idea for further improvement: can you swap using only one temporary variable instead of two as you have now?